sdk/documentintelligence/ai-document-intelligence-rest/CHANGELOG.md

```diff
@@ -2,6 +2,27 @@
 
 # Release History
 
+## 1.1.0 (2025-05-08)
+
+### Features Added
+
+- Supports alternative cloud environments (Azure United States Government and Azure China Cloud). To use an alternative cloud environment, provide a value for the `scopes` field of `DocumentIntelligenceClientOptions#credentials` to configure the client to authenticate within a [Sovereign Cloud](https://learn.microsoft.com/entra/identity-platform/authentication-national-cloud).
+  Import and use `KnownDocumentIntelligenceAudience` to get the correct values for a given cloud environment. The currently supported cloud environments are:
```

Comment:
@xiangyan99 it seems we have similar multi-cloud requirement for data-plane libraries.